src/components/dialogs/manual-filter-creation-dialog.js

```diff
@@ -66,19 +66,18 @@ const ManualFilterRow = ({
                     draggable="true"
                     ref={provided.innerRef}
                     style={{ width: '100%' }}
+                    {...provided.draggableProps}
+                    {...provided.dragHandleProps}
                 >
                     <Grid
                         container
                         item
                         spacing={2}
                         key={index + id + 'container item'}
                         sx={{ width: '100%', height: '50%' }}
-                        draggable="true"
                         ref={provided.innerRef}
-                        {...provided.draggableProps}
-                        {...provided.dragHandleProps}
                     >
-                        <Grid xs={1} item>
+                        <Grid xs={1} item draggable="true">
```

Comment:
Why to use a direct HTML attribute here ?
You seems to misunderstand the use of Draggable component.
Please check this documentation to apply the correct components prop setting
https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/api/draggable.md

Reply:
ok. done.